src/main/java/uk/gov/mint/CTHandler.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.mint;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.store.EntriesUpdateDAO;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+
+public class CTHandler implements Handler {
+    private final CanonicalJsonMapper canonicalJsonMapper;
+    private final String register;
+    private final String ctserver;
+    private final Client client;
+    private final EntryValidator entryValidator;
+
+    public CTHandler(String register, String ctserver, Client client, EntryValidator entryValidator) {
```

Comment:
The client is created via a JerseyClientBuilder object which would require mocking out to do this (alongside mocking the environment and configuration objects too).
Is there a sensible dependency injection framework we could use instead of doing this?